pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.taboola</groupId>
     <artifactId>async-profiler-actuator-endpoint</artifactId>
-    <version>2.0.3_${async.profiler.version}</version>
```

Comment:
Don't need to specify async profiler version as suffix. The version is defined with `io.pyroscope.agent` now